### ISSUE TITLE
Websocket don't build...

### ISSUE THREAD

**Issue by mmardinian** [babelouest/ulfius#59]
Hi,

On your last commit, you did this:
`#if defined(_WIN32) && !defined(U_DISABLE_WEBSOCKET)
  #define U_DISABLE_WEBSOCKET
#endif

#if (MHD_VERSION < 0x00095300) && !defined(U_DISABLE_WEBSOCKET)
  #define U_DISABLE_WEBSOCKET
#endif

#ifndef U_DISABLE_WEBSOCKET
  #ifndef _GNU_SOURCE
    #define _GNU_SOURCE
  #endif
  #include <poll.h>
  #ifndef POLLRDHUP
    #define POLLRDHUP 0x2000
  #endif
#endif
#include <pthread.h>
#include <microhttpd.h>`

So basically you check microhttpd version before including the header file...
Of course MHD_VERSION is not define and Websocket are disabled...

Could you please fix that?

Best regards,

Michael

**Comment by babelouest** [comment]
Sorry :p

Thanks for the issue, I'll change this ASAP

**Comment by babelouest** [comment]
should be fixed now, thanks @mmardinian

**Comment by mmardinian** [comment]
thanks
